lib/ansible/modules/packaging/language/pip.py

```diff
@@ -271,15 +273,22 @@ def _get_packages(module, pip, chdir):
     return (command, out, err)
 
 
-def _is_present(name, version, installed_pkgs, pkg_command):
+def _is_present(module, name, version, installed_pkgs, pkg_command):
     '''Return whether or not package is installed.'''
+    if version is not None:
+        if version[0].isdigit():
+            version = "==%s" % version
+        try:
+            vp = VersionPredicate("%s (%s)" % (name, version))
+        except ValueError:
+            module.fail_json(msg="Can parse invalid version number %s" % version)
```

Comment:
Are there any packages which use abnormale versioning schemes and could be broken here?

Reply:
Just in case user input some typo version here.

Reply:
"Can" or "Cannot"?